A bunch of donut 3 improvements

Camryn-Buttes opened the pull request:
<!-- The text between the arrows are comments - they will not be visible on your PR. -->
<!-- To automatically tag this PR, add the uppercase label(s) surrounded by brackets below, for example: [LABEL] -->
[MAPPING][QOL]
## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->


Makes some needed additions to donut 3

- Adds an oxygen canister to the armory
- Adds a box of tracking implants to security
- Adds an artistic toolbox to the chaplain's office (TO BE CLEAR: this is the toolbox full of paint that spawns in the chapel on manta, oshan, destiny and I think a couple other maps. This is NOT the memetic murder toolbox)
- Adds two biosuit lockers to the pathology lab lobby
- Adds a box of pens to artlab
- Adds a costume eyepatch to the costume wardrobe in the chapel
- Adds a welding supply locker to the research podbay
- Adds a GTM to the maintenance hall south of engineering
- Adds an enucleation spoon to the medical booth
- Adds office supply wall cabinets to the following locations: the nerd dungeon, the public market stall, the bridge, and the qm's office (the little carpeted room they have)
- Adds a lighter and propuff pack to the detective's office
- Adds a portable camera monitor to the detective's office
- Adds a secure briefcase to the detective's office
- Adds a forensics cart to the detective's office, with the following items spawning inside: a box of body bags, two evidence boxes, a luminol spray bottle, an audio recorder, two tapes for the audio recorder, and a hand labeler 


## Why's this needed? <!-- Describe why you think this should be added to the game. -->

donut 3 improvements

## Changelog
<!-- If necessary, put your changelog entry below. Otherwise, please delete it.
Use however you want to be credited in the changelog in place of CodeDude.
Use (*) for major changes and (+) for minor changes. For example: -->

```changelog
(u)Camryn Buttes
(+)Made a bunch of minor improvements and additions to Donut 3.
```

TTerc commented on maps/oshan.dmm:
```diff
@@ -31724,7 +31724,7 @@
 	pixel_x = -6
 	},
-/obj/item/postit_stack,
+/obj/item/item_box/postittit,
```
```suggestion
/obj/item/item_box/postit,
```

TTerc commented on maps/oshan_xmas.dmm:
```diff
@@ -33496,7 +33496,7 @@
 	pixel_x = -6
 	},
-/obj/item/postit_stack,
+/obj/item/item_box/postittit,
```
```suggestion
/obj/item/item_box/postit,
```

TTerc commented on maps/pamgoc.dmm:
```diff
@@ -62862,7 +62862,7 @@
 	pixel_y = 5
 	},
-/obj/item/postit_stack{
+/obj/item/item_box/postittit{
```
```suggestion
/obj/item/item_box/postit{
```

TTerc commented on maps/pamgoc.dmm:
```diff
@@ -63287,7 +63287,7 @@
 	tag = ""
 	},
-/obj/item/postit_stack{
+/obj/item/item_box/postittit{
```
```suggestion
/obj/item/item_box/postit{
```

TTerc commented on maps/pamgoc.dmm:
```diff
@@ -64980,7 +64980,7 @@
 	pixel_y = 7
 	},
-/obj/item/postit_stack{
+/obj/item/item_box/postittit{
```
```suggestion
/obj/item/item_box/postit{
```

Camryn-Buttes commented:
Okay, so I'm trying to merge the suggestions, but it's just throwing an error at me and saying the diff was recently updated and I need to refresh and try again

geneva-breaker commented:
why is this not two PRs

Camryn-Buttes commented:
> why is this not two PRs

Because the old sticky notes kept spawning boxes of new sticky notes on the same tile as the office supply wall cabinets that I added. It was really ugly, so I had to address the issue in SOME way. 

I figured I probably shouldn't half-ass it, so I went ahead and replaced *every* instance of the old sticky notes, except for their definition.

geneva-breaker commented:
> I figured I probably shouldn't half-ass it, so I went ahead and replaced _every_ instance of the old sticky notes, except for their definition.

right, but why is this in the same PR as your assorted donut fixes. 
this puts me in an awkward position of wanting to merge the stickynote fix, but being unable to do so because I'm not trusting "some other miscellaneous improvements" without review. 

And i'm not reviewing Donut3 because it's not a map i'm intimate with.

Camryn-Buttes commented:
> > I figured I probably shouldn't half-ass it, so I went ahead and replaced _every_ instance of the old sticky notes, except for their definition.
> 
> right, but why is this in the same PR as your assorted donut fixes.
> this puts me in an awkward position of wanting to merge the stickynote fix, but being unable to do so because I'm not trusting "some other miscellaneous improvements" without review.
> 
> And i'm not reviewing Donut3 because it's not a map i'm intimate with.

I can provide an exhaustive list of changes later today after I manually push the fix for those postit typos.

Camryn-Buttes commented:
Actually wait, I could just split it into two

Camryn-Buttes commented:
There we go, reverted the postit changes, will be making a separate PR for them

Also, will write up an exhaustive list of changes in a bit.

geneva-breaker commented:
cool and good thank you

Camryn-Buttes commented:
> 
> 
> cool and good thank you

https://github.com/goonstation/goonstation/pull/4115

there we go

Camryn-Buttes commented:
and there's the exhaustive list of changes

Camryn-Buttes commented:
Hrrrgggg fukken merge conflicts

I'll fix em today
